packages/@react-spectrum/color/docs/ColorSlider.mdx

```diff
@@ -0,0 +1,228 @@
+      <pre>Final value: {finalValue.toString('hsl')}</pre>
+    </div>
+  );
+}
+```
+
+## Creating a color picker
+
+### RGBA
+
+This example shows how you could build an RGBA color picker using four color sliders bound to the same
+color value in state. The <TypeLink links={statelyDocs.links} type={statelyDocs.exports.parseColor} />
+function is used to parse the initial color from a hex value, stored in state. The `value` and `onChange` props
+of ColorSlider are used to make the sliders controlled, so that they all update when the color is modified.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('#ff00ff'));
+
+  return (
+    <Flex direction="column">
+      <ColorSlider channel="red" value={color} onChange={setColor} />
+      <ColorSlider channel="green" value={color} onChange={setColor} />
+      <ColorSlider channel="blue" value={color} onChange={setColor} />
+      <ColorSlider channel="alpha" value={color} onChange={setColor} />
+    </Flex>
+  );
+}
+```
+
+### HSLA
+
+This example shows how to build a similar color picker to the one above, using HSLA colors instead.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('hsla(0, 100%, 50%, 0.5)'));
+
+  return (
+    <Flex direction="column">
+      <ColorSlider channel="hue" value={color} onChange={setColor} />
+      <ColorSlider channel="saturation" value={color} onChange={setColor} />
+      <ColorSlider channel="lightness" value={color} onChange={setColor} />
+      <ColorSlider channel="alpha" value={color} onChange={setColor} />
+    </Flex>
+  );
+}
+```
+
+### HSBA
+
+This example shows how to build an HSBA color picker.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('hsba(0, 100%, 50%, 0.5)'));
+  return (
+    <>
+      <ColorSlider channel="hue" value={color} onChange={setColor} />
+      <ColorSlider channel="saturation" value={color} onChange={setColor} />
+      <ColorSlider channel="brightness" value={color} onChange={setColor} />
+      <ColorSlider channel="alpha" value={color} onChange={setColor} />
+    </>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.ColorSlider} links={docs.links} />
+
+## Visual options
+
+### Disabled
+[View guidelines](https://spectrum.adobe.com/page/color-slider/#Disabled)
+
+```tsx example
+<ColorSlider defaultValue="#7f0000"  channel="red" isDisabled />
+```
+
+### Vertical
+[View guidelines](https://spectrum.adobe.com/page/color-slider/#Orientation)
+
+```tsx example
+<ColorSlider defaultValue="#7f0000" channel="red" orientation="vertical" />
+```
+
+### Custom Size
+[View guidelines](https://spectrum.adobe.com/page/color-slider/#Length)
+
+```tsx example
+<Flex direction="column" gap="size-300">
+  <ColorSlider defaultValue="#7f0000" channel="red" orientation="vertical" height={300} />
+  <ColorSlider defaultValue="#7f0000" channel="red" width={300} />
```

Comment:
Maybe add a maxWidth so it doesn't go off the page on mobile?

Reply:
Argh, I always forget to check on mobile, thanks